zmon-common/src/main/java/org/zalando/zmon/domain/CheckDefinition.java

```diff
@@ -88,6 +89,11 @@
     @DatabaseField
     private DefinitionRuntime runtime;
 
+    //Not sure, that this will work
```

Comment:
if you're not sure, maybe PR is not ready? ;)

Reply:
I'm not sure, how this DB entity is filled with data. But since we haven't got any errors, it works fine. Will delete this comment

Reply:
wait a minute, why do you call it a DB field? It should not appear in database and be used only for JSON rendering

Reply:
There is no DTO which is exposed in RESP API. Unfortunately we are exposing the DB entity itself.